From af7b82d6f1e65edcc93d854405fcb46a25adf6c4 Mon Sep 17 00:00:00 2001
From: freedesktop_sdk_updater <freedesktop_sdk_updater@libreml.com>
Date: Fri, 31 May 2024 01:12:54 +0000
Subject: [PATCH] Update elements/components/wayland.bst to 1.23.0-0

Updates elements/components/wayland.bst to 1.23.0-0 (a156431)
Previous version was 1.22.0-0 (b2649cb)
---
 elements/components/wayland.bst | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/elements/components/wayland.bst b/elements/components/wayland.bst
index ab949c1a846..161a14d85cc 100644
--- a/elements/components/wayland.bst
+++ b/elements/components/wayland.bst
@@ -31,7 +31,7 @@ sources:
   track: '*.*.*'
   exclude:
   - '*.*.9*'
-  ref: 1.22.0-0-gb2649cb3ee6bd70828a17e50beb16591e6066288
+  ref: 1.23.0-0-ga156431ea66fe67d69c9fbba8a8ad34dabbab81c
 
 (@):
 - elements/include/cursorpath.yml
-- 
GitLab

